snapm/manager/_manager.py

```
@@ -351,6 +364,205 @@ def _find_mount_point_for_devpath(devpath):
+            bound = None
+        name = None
+        if bound:
+            arg = bound.arguments
+            if arg.get("name") is not None:
+                name = arg["name"]
+            elif arg.get("old_name") is not None:
+                name = arg["old_name"]
+            elif arg.get("uuid") is not None:
+                if isinstance(arg["uuid"], str):
+                    try:
+                        uuid = UUID(u)
```

Ruff (0.12.2)

Undefined name `u`

(F821)